changelogs/unreleased/buildmaster-security-prismjs.yml

```diff
@@ -0,0 +1,3 @@
+description: Fix security alert regarding the prismjs and @babel/runtime package
+change-type: patch
+destination-branches: [master, iso8, iso7]
```

Comment:
you'll have to remove master and iso8. Also not sure if we need to fix them still on iso7 🤔 if it doesn't cause issues ,fine, but if it makes some tests fail, we might not want to spend time fixing them there